bodhi/server/models.py

```diff
@@ -3653,6 +3653,41 @@ def comment_on_test_gating_status_change(target, value, old, initiator):
                 author="bodhi",
             )
 
+    def move_to_stable(self, db):
+        """
+        Move the update to stable if it was set to not spend any time in testing.
```

Comment:
See above comment :-), also the code does more than that in some case it will not request the update to be pushed to stable.